notebooks/dcm/anonymise-dicoms-glob-method.ipynb

```diff
@@ -0,0 +1,95 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from glob import glob\n",
+    "import pathlib\n",
+    "\n",
+    "import pydicom\n",
+    "\n",
+    "from pymedphys.dicom import anonymise_dicom_dataset"
+   ]
+  },
```

Comment:
merged in Jacob's changes to pre-empt any conflicts. But we'll merge his changes in first so they're appropriately attributed to him :)